examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -30,144 +25,237 @@ func Render(_ string) string {
 
-Intuitive and easy to use, gno.land lowers the barrier to web3 and makes
-censorship-resistant platforms accessible to everyone. If you want to help lay
-the foundations of a fairer and freer world, join us today. 
 
-## Learn about gno.land
+**/about /github /try gno**
 
-- [About](/about)
-- [GitHub](https://github.com/gnolang)
-- [Blog](/blog)
-- [Events](/events)
-- Tokenomics (soon)
-- [Partners, Fund, Grants](/partners)
-- [Explore the Ecosystem](/ecosystem)
-- [Careers](https://jobs.ashbyhq.com/allinbits)
+gno.land is a smart contract platform built for Gno, a deterministic implementation of Go, designed for secure, auditable, and reusable decentralized applications (dApps). Unlike historical blockchain development languages, Gno allows Go developers to easily transfer their skills without needing to learn a completely new language. With type safety and composability, gno.land makes building scalable dApps secure, intuitive and reusable.
 
-## Build with Gno
+At its core, gno.land is the ‘GitHub’ of the Gno ecosystem, where transparent, open-source smart contracts are built on top of a native censorship-resistant decentralized operating system (dOS) starting with the Gno Virtual Machine (GnoVM).
 
-- [Write Gno in the browser](https://play.gno.land)
-- [Read about the Gno Language](/gnolang)
-- [Visit the official documentation](https://docs.gno.land)
-- [Gno by Example](https://gno-by-example.com/)
-- [Efficient local development for Gno](https://docs.gno.land/gno-tooling/cli/gno-tooling-gnodev)
-- [Get testnet GNOTs](https://faucet.gno.land)
+## Key Features of Gno
 
-## Explore the universe
+- Open & Transparent: All applications are open-source and fully auditable.
+- Composable: Smart contracts are modular, reusable, and customizable to ease development and extend the ecosystem’s capabilities quickly.
+- Censorship-Resistant: Enables platforms with fair moderation and freedom of communication without a central authority.
+- Developer-Friendly: Intuitive design that reduces onboarding timelines and enables faster iteration.
+- Go-Based & Type Safe: Go developers can transition seamlessly to Gno thanks to the well-established Go language, while benefiting from type safety for security.
 
-- [Discover demo packages](https://github.com/gnolang/gno/tree/master/examples)
-- [Gnoscan](https://gnoscan.io)
-- [Portal Loop](https://docs.gno.land/concepts/portal-loop)
-- [Testnet 4](https://test4.gno.land/)
-- [Faucet Hub](https://faucet.gno.land)
 
-## [Latest Blogposts](/r/gnoland/blog)
+---
 
-:latest-blogposts:
+Use Cases
 
-## [Latest Events](/r/gnoland/events)
+## Why Gno and gno.land?
 
-:upcoming-events:
+gno.land’s smart contract platform is a first step in an experiment to build a complete dOS to bypass the centralized and controlled restrictions of the current Web, and power new social platforms, governance frameworks, and economic models that favor contribution-based incentives.
```

Comment:
2cts: We have an "About" link, followed by an introductory paragraph and here the "why gno" paragraph. 

Could we try create a single more efficient introductory paragraph or set of paragraphs with a link to learn more (/about)?